docs/topics/mta-web-creating-applications.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-applications_{context}"]
+= Creating an application profile
+
+You can create an application profile in order to add an application to the *Application Invcentory* for assessment and analysis.
+
+[TIP]
+====
+Before creating an application profile, it is helpful to set up business services, check tags and tag types, and create additions as needed.
+====
+
+.Procedure
+
+. In *Developer* view, click *Application Inventory*.
+. Click *Create new*.
++
+// ![](/Tackle2/AddingApps/NewAppBasic.png)
++
+. Enter the following information or choose from a list:
+
+* Name
+* Description (optional)
+* Business service
+* Tags (optional; one or more)
+* Comments (optional)
+
+. Click the arrow to the right of *Source Code*.
```

Comment:
The arrow is left of Source Code 
![Screenshot from 2022-11-03 13-14-36](https://user-images.githubusercontent.com/87791967/199669298-edccb516-5ee7-4b58-9c20-d7052cdb0e2c.png)